src/CommunityToolkit.Maui/Extensions/ToastNotificationExtensions.windows.cs

```diff
@@ -8,19 +8,6 @@ namespace CommunityToolkit.Maui.Extensions;
 /// </summary>
 public static class ToastNotificationExtensions
 {
-	/// <summary>
-	/// Build Toast Text Notification
-	/// </summary>
-	/// <param name="text">Notification text</param>
-	/// <returns><see cref="XmlDocument"/></returns>
-	public static XmlDocument BuildToastNotificationContent(string text)
```

Comment:
FYI - this is a breaking change, removing a public API

We'll need to bump our Major version number for our next release (which'll hopefully coincide with the release of `MediaPlayer`)

Reply:
That being said, I'd be surprised if removing this API impacts any devs

Reply:
How start coding